PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -28,20 +31,17 @@ class PayPalClient internal constructor(
     private val braintreeClient: BraintreeClient,
     private val internalPayPalClient: PayPalInternalClient = PayPalInternalClient(braintreeClient),
     private val merchantRepository: MerchantRepository = MerchantRepository.instance,
+    getReturnLinkTypeUseCase: GetReturnLinkTypeUseCase = GetReturnLinkTypeUseCase(merchantRepository),
     private val getReturnLinkUseCase: GetReturnLinkUseCase = GetReturnLinkUseCase(merchantRepository),
-    private val getAppSwitchUseCase: GetAppSwitchUseCase = GetAppSwitchUseCase(AppSwitchRepository.instance)
+    private val getAppSwitchUseCase: GetAppSwitchUseCase = GetAppSwitchUseCase(AppSwitchRepository.instance),
+    analyticsParamRepository: AnalyticsParamRepository = AnalyticsParamRepository.instance
```

Comment:
Can these all be private?

Reply:
`getReturnLinkTypeUseCase` and `analyticsParamRepository` are not properties on the class and are just params pass into the constructor. Since they're only used in the `init` functions, they don't need to be defined as class properties.